### ISSUE TITLE
build st-model from training data or gallery data?

### ISSUE THREAD

**Issue by PonyYeh** [Wanggcong/Spatial-Temporal-Re-identification#23]
Thank you for your work!
I have a question want to confirm:
Did you use training data that contains 751 people to build the st-model in [gen_st_model_market.py,](https://github.com/Wanggcong/Spatial-Temporal-Re-identification/blob/master/gen_st_model_market.py#L49) instead of using the gallery data(750 people) to build st-model? 
thank you~

**Comment by Wanggcong** [comment]
Yes, only training data (751 persons) are used to build the st-model in gen_st_model_market.py.
